ee/packages/license/src/validation/validateLicenseLimits.ts

```diff
@@ -9,10 +9,11 @@ export async function validateLicenseLimits(
 	this: LicenseManager,
 	license: ILicenseV3,
 	behaviorFilter: (behavior: LicenseBehavior) => boolean,
+	limitFilter: (limit: LicenseLimitKind) => boolean,
 ): Promise<BehaviorWithContext[]> {
 	const { limits } = license;
 
-	const limitKeys = Object.keys(limits) as (keyof ILicenseV3['limits'])[];
+	const limitKeys = (Object.keys(limits) as LicenseLimitKind[]).filter(limitFilter || (() => true));
```

Comment:
since `limitFilter` is not optional, is the || needed?
btw, maybe it can also be as a default value for the param 👀